packages/@aws-cdk/aws-rds/lib/cluster.ts

```diff
@@ -141,6 +142,34 @@ export interface DatabaseClusterProps {
    * @default - A role is automatically created for you
    */
   readonly monitoringRole?: IRole;
+
+  /**
+   * Role that will be associated with this DB cluster to enable S3 import through the LOAD DATA FROM S3 command
+   *
+   * @default - A role is created for you if the s3ImportBuckets property is set
+   */
+  readonly s3ImportRole?: IRole;
```

Comment:
All these properties are only applicable to Aurora, IIUC. 

It would be worth adding a statement to this effect here, and validations in the implementation.

Reply:
Done.  BTW I do agree with #6713.  Will make things quite cleaner.